lib/data_pull.rb

```diff
@@ -263,4 +266,46 @@ def run(args:, config:)
       )
     end
   end
+
+  class ProfileSummary
+    def run(args:, config:)
+      uuids = args
+
+      users = User.includes(:profiles).where(uuid: uuids).order(:uuid)
+
+      table = []
+      table << %w[uuid current_status activated_timestamp disabled_reason fraud_review_pending_timestamp fraud_rejection_timestamp]
+
+      users.each do |user|
+        if user.profiles.any?
+          user.profiles.sort_by(&:id).each do |profile|
+            profile_status = profile.active ? 'Active' : 'Inactive'
+            table << [
+              user.uuid,
+              profile_status,
+              profile.activated_at,
+              profile.deactivation_reason,
+              profile.fraud_review_pending_at,
+              profile.fraud_rejection_at,
+            ]
+          end
+        elsif config.include_missing?
+          table << [user.uuid, '[HAS NO PROFILE]', nil, nil, nil, nil]
+        end
+      end
+
+      if config.include_missing?
+        (uuids - users.map(&:uuid)).each do |missing_uuid|
+          table << [missing_uuid, '[UUID NOT FOUND]', nil, nil, nil, nil]
```

Comment:
ditto for removing extras
```suggestion
          table << [missing_uuid, '[UUID NOT FOUND]']
```